feature/google/src/main/java/com/revenuecat/purchases/google/BillingWrapper.kt

```diff
@@ -467,22 +467,31 @@ class BillingWrapper(
         }
     }
 
-    internal fun getPurchaseType(purchaseToken: String): ProductType {
+    @VisibleForTesting(otherwise = PRIVATE)
+    @Suppress("ReturnCount")
```

Comment:
I actually meant to disable this check globally, since I personally believe it doesn't offer that much value. Happy to hear other people's thoughts though.

Reply:
yeah i'm totally fine with that..

Reply:
looks like globally disabling a check requires a separate config file. ok if we do that in a different PR?

Reply:
Sure!